Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPostCampaignViewControllerTests.swift

```diff
@@ -83,122 +90,6 @@ final class NoShippingPostCampaignViewControllerTests: TestCase {
     }
   }
 
-  func testView_PledgeContext_FixPaymentMethod_ErroredCard() {
-    let response = UserEnvelope<GraphUser>(me: self.userWithCards)
-    let mockService = MockService(fetchGraphUserResult: .success(response))
-    let project = Project.template
-      |> Project.lens.state .~ .successful
-      |> Project.lens.personalization.isBacking .~ true
-      |> Project.lens.personalization.backing .~ (
-        .template
-          |> Backing.lens.paymentSource .~ Backing.PaymentSource.visa
-          |> Backing.lens.status .~ .errored
-          |> Backing.lens.reward .~ Reward.noReward
-          |> Backing.lens.rewardId .~ Reward.noReward.id
-          |> Backing.lens.amount .~ 695.0
-          |> Backing.lens.bonusAmount .~ 695.0
-          |> Backing.lens.shippingAmount .~ 0
-      )
-    let mockConfigClient = MockRemoteConfigClient()
-    mockConfigClient.features = [
-      RemoteConfigFeature.noShippingAtCheckout.rawValue: true
-    ]
-
-    orthogonalCombos([Language.en], [Device.phone4_7inch]).forEach { language, device in
-      withEnvironment(
-        apiService: mockService,
-        currentUser: User.template,
-        language: language,
-        remoteConfigClient: mockConfigClient
-      ) {
-        let controller = NoShippingPostCampaignCheckoutViewController.instantiate()
-        let reward = Reward.noReward
-        let data = PostCampaignCheckoutData(
-          project: project,
-          baseReward: reward,
-          rewards: [reward],
-          selectedQuantities: [:],
-          bonusAmount: 0,
-          total: 5,
-          shipping: nil,
-          refTag: nil,
-          context: .pledge,
-          checkoutId: "0",
-          backingId: "backingId",
-          selectedShippingRule: .template
-        )
-        controller.configure(with: data)
-        let (parent, _) = traitControllers(device: device, orientation: .portrait, child: controller)
-        parent.view.frame.size.height = 800
-
-        self.scheduler.advance(by: .seconds(1))
-
-        self.allowLayoutPass()
-
-        assertSnapshot(
-          matching: parent.view,
-          as: .image(perceptualPrecision: 0.98),
-          named: "lang_\(language)_device_\(device)"
-        )
-      }
-    }
-  }
-
-  func testView_ShowsShippingSummaryViewSection() {
```

Comment:
All tests show the shipping summary, so deleted because this test doesn't add value